distribution/std-lib/Standard/src/Visualization/Histogram.enso

```diff
@@ -0,0 +1,70 @@
+from Standard.Base import all
```

Comment:
As a general comment for this file, you could define most of these as extension methods which would be much more idiomatic.